web/src/main/java/com/futurewei/alcor/web/entity/dataplane/NeighborInfo.java

```diff
@@ -0,0 +1,42 @@
+package com.futurewei.alcor.web.entity.dataplane;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.Data;
+
+@Data
+public class NeighborInfo {
+
+    public String getHostId() {
+        return hostId;
+    }
+
+    public void setHostId(String hostId) {
+        this.hostId = hostId;
+    }
+
+    public String getPortId() {
+        return portId;
+    }
+
+    public void setPortId(String portId) {
+        this.portId = portId;
+    }
+
+    @JsonProperty("host_ip")
+    private String hostIp;
+
+    @JsonProperty("host_id")
+    private String hostId;
+
+    @JsonProperty("port_id")
```

Comment:
Do you think making this as a list is a good idea? 

May be it will help us group by host later.